includes/Plugin.php

```diff
@@ -327,7 +327,7 @@ public function register() {
 		add_action( 'widgets_init', [ $this, 'register_widgets' ] );
 
 		$tinymce = new TinyMCE();
-		add_action( 'current_screen', [ $tinymce, 'init' ] );
+		add_action( 'admin_init', [ $tinymce, 'init' ] );
```

Comment:
There was a reason we put this on current current screen. Is the screen object setup at admin_init?

Reply:
In my testing, `current_screen` didn't work as expected. No TinyMCE button was displayed at all. That's because `$screen->is_block_editor()` returned true.

What I did:

1. Active Classic Editor plugin and make classic editor the default
2. Create a new post (will open classic editor)
3. See that there is no TinyMCE button


But you're right that `admin_init` might not work either.

I think we have to revert the previous change to using `current_screen` / `$this->is_block_editor()` in `init`. It does not work as expected.

Reply:
Will fix in follow-up PR.